### PR TITLE
Update nodejs data for `Blob`

### PR DESCRIPTION
**Pull request by skyclouds2001**
#### Summary

https://nodejs.org/api/globals.html#class-blob
https://nodejs.org/api/buffer.html#class-blob

### CODE REVIEW

File: api/Blob.json [hunk 1]
```diff
@@ -328,7 +364,7 @@
               "version_added": false
             },
             "nodejs": {
-              "version_added": "15.7.0"
+              "version_added": "16.7.0"
```
Comment (caugner) [inline api/Blob.json:367]: Good catch!

**Review by caugner — COMMENTED** [review]
Overall LGTM, just one question.

File: api/Blob.json [hunk 2]
```diff
@@ -82,9 +84,16 @@
               "version_added": "10",
               "notes": "IE10 throws <code>InvalidStateError</code> with a <code>TypedArray</code> argument. Consider using <code>MSBuilder</code> as an alternative."
             },
-            "nodejs": {
-              "version_added": "15.7.0"
-            },
+            "nodejs": [
+              {
+                "version_added": "15.7.0",
+                "notes": "Before v16.7.0, the <code>endings</code> option use a non-standard implement."
```
Comment (skyclouds2001) [inline api/Blob.json:90]:
```suggestion
                "notes": "Before v16.7.0, the <code>endings</code> option represents the character encoding to use for string sources, not the line endings in string source parts per the specification."
```

[the remark this answers is not in the document]
Comment (skyclouds2001) [reply api/Blob.json]: https://nodejs.org/docs/v16.17.0/api/buffer.html#new-bufferblobsources-options
https://nodejs.org/docs/v15.7.0/api/buffer.html#buffer_new_buffer_blob_sources_options

File: api/Blob.json [hunk 3]
```diff
@@ -87,7 +87,7 @@
             "nodejs": [
               {
                 "version_added": "15.7.0",
-                "notes": "Before v16.7.0, the <code>endings</code> option use a non-standard implement."
+                "notes": "Before v16.7.0, the <code>endings</code> option represents the character encoding to use for string sources, not the line endings in string source parts per the specification."
```
Comment (caugner) [inline api/Blob.json:90]: I think this isn't quite right. The History in the Node.js docs states:

> v16.7.0: Added the standard `endings` option to replace line-endings, and removed the non-standard `encoding` option.

This states two things:
1. The `endings` option was only added in v16.7.0.
2. Before v16.7.0, a non-standard `encoding` option was supported (but as it's non-standard I wouldn't mention it).

So I would suggest to put `partial_implementation: true` for this range, mentioning that the `endings` option was not supported:

```suggestion
                "notes": "Before v16.7.0, the <code>endings</code> option was not supported."
```

**Review by caugner — DISMISSED** [review]

File: browsers/nodejs.json [hunk 4]
```diff
@@ -620,11 +634,18 @@
           "release_notes": "https://nodejs.org/en/blog/release/v21.7.0",
           "status": "retired",
           "engine": "V8",
-          "engine_version": "11.8"
+          "engine_version": "12.4"
```
Comment (skyclouds2001) [inline browsers/nodejs.json:637]:
```suggestion
          "engine_version": "11.8"
```

File: api/Blob.json [hunk 5]
```diff
@@ -82,9 +84,16 @@
               "version_added": "10",
               "notes": "IE10 throws `InvalidStateError` with a `TypedArray` argument. Consider using `MSBuilder` as an alternative."
             },
-            "nodejs": {
-              "version_added": "15.7.0"
-            },
+            "nodejs": [
+              {
+                "version_added": "15.7.0",
+                "notes": "Before v16.7.0, the <code>endings</code> option was not supported."
```
Comment (queengooborg) [inline api/Blob.json:90]:
```suggestion
                "notes": "Before v16.7.0, the `endings` option was not supported."
```

**Review by queengooborg — CHANGES_REQUESTED** [review]

**Review by queengooborg — APPROVED** [review]

**Pull request merged**